src/buttons/button-radio.directive.ts

```diff
@@ -22,41 +23,53 @@ export const RADIO_CONTROL_VALUE_ACCESSOR: any = {
 export class ButtonRadioDirective implements ControlValueAccessor, OnInit {
   onChange: any = Function.prototype;
   onTouched: any = Function.prototype;
+  private _value: any;
 
   /** Radio button value, will be set to `ngModel` */
   @Input() btnRadio: any;
   /** If `true` — radio button can be unchecked */
   @Input() uncheckable: boolean;
   /** Current value of radio component or group */
-  @Input() value: any;
+  @Input() get value(): any {
+    return this.group ? this.group.value : this._value;
```

Comment:
`return this.group ? this.group.value : this._value;`
try
`return this.group?.value || this._value;`

Reply:
Not supported by TS yet https://github.com/Microsoft/TypeScript/issues/16